tests/test_step_methods.py

```diff
@@ -557,8 +560,15 @@ def test_HSStep_NegativeBinomial():
         eta = pm.NegativeBinomial("eta", mu=beta.dot(X.T), alpha=1, shape=N)
         pm.Normal("y", mu=at.exp(eta), sigma=1, observed=y_nb)
 
-        with pytest.raises(NotImplementedError):
+        with pytest.raises(SamplingError):
```

Comment:
Was this change due to an update in PyMC3?

Reply:
> Looks like there's still an issue with one of the tests.

Yes, I think only the NB portion is fixed. Sorry. 


> Was this change due to an update in PyMC3?
I am not sure,  But this was now samplingError is raised instead of NotImplementedError ones. How should I check ?

Reply:
The reason that it switched from `NotImplementedError` to `samplingError` is because I updated the logic for finding design matrix. In the previous case if the model is set up as in the test we were not able to find the design matrix. Thus the `NotImplementedError`.